src/java/picard/illumina/IlluminaBasecallsToFastq.java

```diff
@@ -68,15 +68,32 @@
 import java.util.Set;
 
 @CommandLineProgramProperties(
-        usage = "Generate fastq file(s) from data in an Illumina basecalls output directory.\n" +
-                "Separate fastq file(s) are created for each template read, and for each barcode read, in the basecalls.\n" +
-                "Template fastqs have extensions like .<number>.fastq, where <number> is the number of the template read,\n" +
-                "starting with 1.  Barcode fastqs have extensions like .barcode_<number>.fastq, where <number> is the number\n" +
-                "of the barcode read, starting with 1.",
-        usageShort = "Generate fastq file(s) from data in an Illumina basecalls output directory",
-        programGroup = Illumina.class
+  usage = IlluminaBasecallsToFastq.USAGE_SUMMARY + IlluminaBasecallsToFastq.USAGE_DETAILS,
+  usageShort = IlluminaBasecallsToFastq.USAGE_SUMMARY,
+  programGroup = Illumina.class
 )
 public class IlluminaBasecallsToFastq extends CommandLineProgram {
+  static final String USAGE_SUMMARY = "Generate fastq file(s) from Illumina basecall read data.  ";
+  static final String USAGE_DETAILS = "<p>This tool generates fastq files from data in an Illumina basecalls (*.bcl) output directory.  " +
+          "Separate fastq files are created for each template, barcode, and index read.   In the absence of sample pooling (multiplexing) and/or barcodes, then an OUTPUT_PREFIX (file) must be provided as the sample identifier.  For multiplexed samples, a MULTIPLEX_PARAMS (file) must be used.  The MULTIPLEX_PARAMS file contains the list of sample barcodes used to sort template, barcode, and index reads.  It is essentially the same as the BARCODE_FILE used in the" +
```

Comment:
Re: OUTPUT_PREFIX and File vs String, what do you mean by "I have fixed this"? It's declared correctly as `public String OUTPUT_PREFIX;` in the tool code as far as I can tell.